### DISCUSSION TITLE
[Kavita] Issues count number

### DISCUSSION THREAD

**Discussion by DieselTech** [Kareadita/Kavita#2560] [Ideas]
### Idea Description

Posted by ZeroSkrull
· 8 months ago
· 12 Upvotes
https://feats.kavitareader.com/posts/354/issues-count-number

Description

Would be great if the series have a number of how many issues have. Thanks.

### Duration of Using Kavita

42

### Idea Category

Feature Enhancement

### Agreement

- [X] I agree that this is solely for submitting ideas, and I will search for existing ideas before posting.

**Comment by majora2007** [comment]
Doesn't Kavita already do this? 
<img width="292" height="94" alt="image" src="https://github.com/user-attachments/assets/07a0f8e4-312f-4431-88ef-b0dd2748c228" />
